### PR TITLE
chore(github issue): fit with nodejs.dev

### PR DESCRIPTION
**Pull request by AugustinMauroy**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by ovflowd — APPROVED** [review]

**Comment by ovflowd** [comment]
Looks good, but let's merge this after #4991 gets merged. I can't deal with rebasing anymore lol

**Pull request closed without merging**
